Src/LinearSolvers/MLMG/AMReX_MLCurlCurl.cpp

```diff
@@ -228,7 +230,9 @@ void MLCurlCurl::interpolation (int amrlev, int fmglev, MF& fine,
             }
         });
     }
-    Gpu::streamSynchronize();
```

Comment:
We should either move the sync into the for-loop or make `MultiFab cfine` an array.